### PR TITLE
fix(participants-pane): Use selector for displaying chat context item

### PR DESCRIPTION
**Pull request by gimre**

### CODE REVIEW

File: react/features/base/config/functions.web.js [hunk 1]
```diff
@@ -44,3 +44,14 @@ export function getToolbarButtons(state: Object): Array<string> {
 
     return Array.isArray(toolbarButtons) ? toolbarButtons : TOOLBAR_BUTTONS;
 }
+
+/**
+ * Curried selector to check if the specificied button is enabled.
```
Comment (quitrk) [inline react/features/base/config/functions.web.js:49]: typo

Comment (gimre) [reply react/features/base/config/functions.web.js:49]: can you be more specificic?

Comment (saghul) [reply react/features/base/config/functions.web.js:49]: "Specified"

Comment (gimre) [reply react/features/base/config/functions.web.js:49]: haha :D

**Review by quitrk — APPROVED** [review]

**Pull request merged**